docs/source/workflows/performance_workflows.rst

```diff
@@ -73,8 +73,10 @@ For numeric responses, performance metrics include root mean squared error (RMSE
    Residuals are calculated using ``(Predicted - Actual)/(Uncertainty Estimate)``.
    A value below 1 indicates the model is underconfident, i.e. actual values are within predicted error bars, on average.
    A value over 1 indicates the model is overconfident, i.e. actual values fall outside predicted error bars, on average.
--  Standard confidence is the fraction of actual values within the predicted error bars.
-   (0.68 is perfectly calibrated.)
+-  Coverage probability is the fraction of observations for which the magnitude of the error is within a confidence interval of a given coverage level.
```

Comment:
Confidence interval or prediction interval?  With cross-validation, I think they are prediction.

Reply:
You're right. Fixed.